src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -58,7 +58,7 @@
 								</div>
 							</td>
 							<td class="deletedAt">
-								<Moment class="timestamp" :timestamp="item.deletedAt" />
+								<Moment class="timestamp" :timestamp="toTimestamp(item.deletedAt)" />
```

Comment:
Might as well configure the `<Moment>` component to use Date types instead of timestamps, and then invert the convert logic.